hackyourrepo-app/script.js

```diff
@@ -1,5 +1,202 @@
+repoUpdatedDivEl.appendChild(repoUpParagraphEl);
+
+contributorsSectionEl.appendChild(contributorsDivEl);
+contributorsDivEl.appendChild(contributorsParagraphEl);
+
+// Create DOM elements: Footer
+const footerEl = document.createElement('footer');
+footerEl.classList.add('footer-container');
+const footerParagraphEl = document.createElement('p');
+footerParagraphEl.innerText = 'by Danny Osorio';
+
+const myGithubLinkEl = document.createElement('a');
+myGithubLinkEl.href = 'https://github.com/dyosorio';
+myGithubLinkEl.target = '_blank';
+const myGithubItemEl = document.createElement('i');
+myGithubItemEl.classList.add('fab');
+myGithubItemEl.classList.add('fa-github');
+
+const myCodepenLinkEl = document.createElement('a');
+myCodepenLinkEl.href = 'https://codepen.io/danny-osorio';
+myCodepenLinkEl.target = '_blank';
+const myCodepenItemEl = document.createElement('i');
+myCodepenItemEl.classList.add('fab');
+myCodepenItemEl.classList.add('fa-codepen');
+
+// Add elements to the DOM: Footer
+document.body.appendChild(footerEl);
+footerEl.appendChild(footerParagraphEl);
+footerParagraphEl.appendChild(myGithubLinkEl);
+myGithubLinkEl.appendChild(myGithubItemEl);
+footerParagraphEl.appendChild(myCodepenLinkEl);
+myCodepenLinkEl.appendChild(myCodepenItemEl);
+
+function main() {
+  // create fetch function
+  function fetchData(url) {
+    return fetch(url)
+      .then(response => response.json())
+      .catch(error => {
+        console.log(error);
+        mainEl.innerHTML = `<div class="error-message">${error}</div>`;
+      });
+  }
+
+  // add repoData to the select menu when the select element is clicked
+  selectEl.addEventListener('click', () => {
+    fetchData(url).then(data => {
+      console.log(data);
+      addRepoNamesToSelectMenu(data); // put this data into the DOM
+    });
+  });
+
+  // generate the repo names to feed the select tag
+  function addRepoNamesToSelectMenu(data) {
+    // grab array of objects
+    const repoArray = data;
+    console.log(repoArray);
+
+    for (let index = 0; index < repoArray.length; index++) {
+      const repoName = repoArray[index].name;
+      const repOptionEl = document.createElement('option');
+      repOptionEl.innerHTML = `
+            <option value="${index}">${repoName}</option>
+        `;
+      selectEl.appendChild(repOptionEl);
+    }
+  }
+
+  // The 'change' event is fired when an alteration to the <select> element's value is committed by the user.
+  selectEl.addEventListener('change', event => {
+    console.log(event.target.value); // the selected option value
+
+    fetchData(
+      'https://api.github.com/orgs/HackYourFuture/repos?per_page=100',
+    ).then(data => {
+      // loop through the data and match the select value with a repo name inside the data
+      for (let index = 0; index < data.length; index++) {
+        contributorsSectionEl.appendChild(contributorCard);
+        const repoData = data[index];
+        if (event.target.value === repoData.name) {
```

Comment:
```suggestion
const repoData = data.find(el => el.name == event.target.value)
```
Did you know you can use find to find an element in an array?